### PR TITLE
add option to change tab width

### PR DESCRIPTION
**Pull request by rruiter87**
Added it for me, maybe you'd like it as well?

### CODE REVIEW

**Review by klauer — COMMENTED** [review]
Thanks for the PR @rruiter87 - I think it's nice to have this be customizable.

~Looking back, two spaces probably would have been a good choice starting out. Since we have 4 spaces for all of our projects at the moment, though, reformatting every xml file might be a bit much.~

Edit: on second thought, my morning brain had me confused with our Python source settings. 2 has been the default. Sorry about that!

Roping in @ZLLentz to get his input before jumping ahead.

File: pre_commit_hooks/xml_format.py [hunk 1]
```diff
@@ -34,6 +36,7 @@ def main(args=None):
     if args is None:
         parser = argparse.ArgumentParser()
         parser.add_argument('filenames', nargs='*')
+        parser.add_argument('--tab-width', type=int, default=TAB_WIDTH)
```
Comment (ZLLentz) [inline pre_commit_hooks/xml_format.py:39]: Hi, thanks for the contribution!

However, I'm confused, is there a commit missing here? This argparse argument you added isn't used below in the call to `fix_file`.

Comment (rruiter87) [reply pre_commit_hooks/xml_format.py:39]: Good catch! There is a missing commit indeed. I noticed the argument was missing when I was testing it locally, but forgot to push it 😅

Comment (ZLLentz) [reply pre_commit_hooks/xml_format.py:39]: I figured as much, it happens all the time! I'm going to go ahead and merge this.
Thanks again for the pull request!

**Pull request merged**